index.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    ::before
+    <div class="container">
+      <header class="header">
+        <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
+    <p id="discription" class="discription text-center">Thank you for taking the time to help us improve the platform</p>
+      </header>
+    
+    <form id="form">
+      <div class="form-group">
+        <label for="name">Name</label>
+      <input type="text" name="name" placeholder="Enter your name" class="form-control" required/> 
+      </label>
+      </div>
+
+      <div class="form-group">
+        <label for="email">Email</label>
+      <input type="text" name="name" placeholder="Enter your name" class="form-control" required/> 
+      </label>
+      </div>
+
+      
+
+      <div class="form-group">
```

Comment:
I think it would be better if you could remove all unsed empty lines from your code as this will make your code more readable and clean.👍😎